crates/tests/hstring/tests/display.rs

```diff
@@ -0,0 +1,8 @@
+use windows::core::*;
+
+#[test]
+fn display() {
```

Comment:
There's a whole bunch of [tests for `HSTRING` in the `test_core` crate](https://github.com/microsoft/windows-rs/blob/d4bb98da9a6188927026ee81137216eeea8da4e1/crates/tests/core/tests/hstrings.rs) (which could just be tests in the `windows` crate itself TBH). I don't think we need separate crate for this.